.github/scripts/setup.sh

```diff
@@ -53,20 +51,7 @@ sudo apt-get -qq install --no-install-recommends --allow-unauthenticated -yy \
 echo "tester ALL=(root) NOPASSWD:ALL" | sudo tee /etc/sudoers.d/tester
 sudo chmod 0440 /etc/sudoers.d/tester
 
-(
-    cd /tmp/ || exit 1
```

Comment:
`install-bitcoind.sh` is missing the `|| exit 1` on this; not sure if it matters?

Reply:
All good: install-bitcoin.sh correctly does `set -e` which means "exit on any unchecked failure".